### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by nahansans**

### CODE REVIEW

File: react-native-firebase-messaging-and-react-native-push-notification.md/readme.md [hunk 1]
```diff
@@ -3,4 +3,52 @@
...
+
+# Install the messaging module
+npm i @react-native-firebase/messaging
+
+# Install Push Notification
+npm install --save react-native-push-notification
+
+# If you're developing your app using iOS, run this command
+cd ios/ && pod install
+```
```
Comment (reyn-nova) [inline react-native-firebase-messaging-and-react-native-push-notification.md/readme.md:21]: kayake mesih ada nif, coba lihat dari project realtime chat mu, nambah di build.gradle, app/build.gradle, android-manifest.xml dan strings.xml

Comment (nahansans) [reply react-native-firebase-messaging-and-react-native-push-notification.md/readme.md:21]: yang rn message kayake gak ada konfigurasi gradle, cuman yang push notif aja kayake, wait tak cek e

Comment (reyn-nova) [reply react-native-firebase-messaging-and-react-native-push-notification.md/readme.md:21]: build.gradle: https://github.com/nahansans/realtime_chat/blob/ed1d9c5dd4975bdf078d0c8aa62d57fabc958652/android/build.gradle#L16

app/build.gradle: https://github.com/nahansans/realtime_chat/blob/ed1d9c5dd4975bdf078d0c8aa62d57fabc958652/android/app/build.gradle#L222

**Comment by reyn-nova** [comment]
😃

Screen Shot 2020-10-23 at 10 38 18

**Pull request merged**